test/spec/adapters/lifestreet_spec.js

```diff
@@ -44,7 +47,7 @@ describe ('LifestreetAdapter', () => {
 
       beforeEach(() => {
         tagRequests = [];
-        request = utils.extend(request, BIDDER_REQUEST);
+        request = copy(BIDDER_REQUEST);
```

Comment:
for whatever reason, just replacing `utils.extend` with `Object.assign` was causing the tests to fail.  I think they were built in away that made them dependent on the broken `utils.extend` functionality.  I didn't do a deep-dive to figure out why they were failing, but I noticed that the intent was to have a fresh `request` object before each unit test; `copy` worked without breaking the tests.